recipes/gnulib/all/conanfile.py

```diff
@@ -1,28 +1,28 @@
 from conans import ConanFile, tools
 import os
-import re
 import shutil
 
+required_conan_version = ">=1.33.0"
+
 
 class GnuLibConanFile(ConanFile):
     name = "gnulib"
     description = "Gnulib is a central location for common GNU code, intended to be shared among GNU packages."
     homepage =  "https://www.gnu.org/software/gnulib/"
     url = "https://github.com/conan-io/conan-center-index"
-    topics = ("conan", "gnulib", "library", "gnu")
+    topics = ("gnulib", "library", "gnu")
     license = ("GPL-3.0-or-later", "LGPL-3.0-or-later", "Unlicense")
-    no_copy_source = True
 
-    # Added to test on CI
-    settings = "os_build", "arch_build"
```

Comment:
Keep settings, this package is pre-built.

Reply:
Yes, it's header-only like.
It only contains bash/python scripts.